src/ArcGateway/ArcGateway.Autorest/docs/New-AzArcGateway.md

```diff
@@ -0,0 +1,332 @@
+---
+external help file:
+Module Name: Az.ArcGateway
+online version: https://learn.microsoft.com/powershell/module/az.arcgateway/new-azarcgateway
+schema: 2.0.0
+---
+
+# New-AzArcGateway
+
+## SYNOPSIS
+The operation to create or update a gateway.
+
+## SYNTAX
+
+### CreateExpanded (Default)
+```
+New-AzArcGateway -Name <String> -ResourceGroupName <String> -Location <String> [-SubscriptionId <String>]
+ [-GatewayType <String>] [-Tag <Hashtable>] [-DefaultProfile <PSObject>] [-AsJob] [-NoWait] [-Confirm]
+ [-WhatIf] [<CommonParameters>]
+```
+
+### Create
+```
+New-AzArcGateway -Name <String> -ResourceGroupName <String> -Parameter <IGateway> [-SubscriptionId <String>]
+ [-DefaultProfile <PSObject>] [-AsJob] [-NoWait] [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+### CreateViaIdentity
+```
+New-AzArcGateway -InputObject <IArcGatewayIdentity> -Parameter <IGateway> [-DefaultProfile <PSObject>]
+ [-AsJob] [-NoWait] [-Confirm] [-WhatIf] [<CommonParameters>]
+```
+
+### CreateViaIdentityExpanded
```

Comment:
You may don't need this parameterset

Reply:
we still want to keep this parameter set in case of use.